Extension/Global/Helpers.cs

```diff
@@ -31,5 +31,18 @@ public static string AsString(this SortMode sortByOption)
                 default: throw new ArgumentOutOfRangeException(nameof(sortByOption), sortByOption, null);
             }
         }
+
+        public static string AsString(this TypeSortOption typeSortOption)
+        {
+            switch (typeSortOption)
+            {
+                // TODO: Add more options
+                case TypeSortOption.CAVALRY: return "Cavalry";
+                case TypeSortOption.RANGED_CAVALRY: return "Ranged Cavalry";
+                case TypeSortOption.INFANTRY: return "Infantry";
+                case TypeSortOption.RANGED: return "Archers";
```

Comment:
What about crossbowmen?

Reply:
Can add more types later, however crossbows are covered by the `IsArcher` property in the current type comparer